packages/code-du-travail-frontend/pages/_app.js

```diff
@@ -64,17 +66,21 @@ export default class MyApp extends App {
     const { Component, pageProps } = this.props;
     if (pageProps.statusCode) {
       return (
-        <ThemeProvider theme={theme.colors}>
-          <GlobalStyles />
-          <ErrorPage statusCode={pageProps.statusCode} />
+        <ThemeProvider>
+          <div>
```

Comment:
Fragment are not allowed as theme provider child ?

Reply:
The styled component ThemeProvider only accepts a single child and fragment is not accepted

Reply:
thanks !

Reply:
Yep because in the end there is node html behind Routes and global styles

Reply:
I'm wrong, tested